client/my-sites/stats/stats-notices/jitm-wrapper.tsx

```diff
@@ -0,0 +1,30 @@
+import clsx from 'clsx';
+import AsyncLoad from 'calypso/components/async-load';
+import { useSelector } from 'calypso/state';
+import { getTopJITM } from 'calypso/state/jitm/selectors';
+import { StatsNoticeProps } from './types';
+
+const JITMWrapper: React.FC< StatsNoticeProps > = ( { isOdysseyStats } ) => {
+	const messagePath = isOdysseyStats
+		? 'wp:jetpack_page_stats:admin_notices'
```

Comment:
Do we need another change on Jetpack to identify JITM messages for these two cases?

Reply:
No we don't need any. The message paths are kept as what they are to be consistent.

https://github.com/Automattic/jetpack/blob/875d6d75fac08bcbb6262bc185f6dbc73e497c4a/projects/packages/jitm/src/class-jitm.php#L218

https://github.com/Automattic/wp-calypso/blob/fbc1c859e71b558c1388382fb59b8ae705a2d849/client/layout/index.jsx#L392